core/src/franchise/player/player.controller.ts

```diff
@@ -154,4 +166,77 @@ export class PlayerController {
 
         return "SUCCESS";
     }
+
+    @MessagePattern(CoreEndpoint.GetPlayerByPlatformId)
+    async getPlayerByPlatformId(@Payload() payload: unknown): Promise<CoreOutput<CoreEndpoint.GetPlayerByPlatformId>> {
+        const data = CoreSchemas[CoreEndpoint.GetPlayerByPlatformId].input.parse(payload);
+
+        const game = await this.gameService.getGameById(data.gameId);
```

Comment:
We're only using `game.id`, which we have on `data.gameId`. Fetching `game` is unnecessary